test/package.json

```diff
@@ -17,20 +17,21 @@
     "@bedrock/core": "^6.0.1",
     "@bedrock/express": "^8.0.0",
     "@bedrock/https-agent": "^4.0.0",
-    "@bedrock/jsonld-document-loader": "^3.0.0",
-    "@bedrock/karma": "^5.1.0",
+    "@bedrock/jsonld-document-loader": "^4.0.0",
+    "@bedrock/karma": "^6.0.0",
     "@bedrock/mongodb": "^10.0.0",
     "@bedrock/passport": "^11.0.0",
-    "@bedrock/security-context": "^7.0.0",
+    "@bedrock/security-context": "^8.0.0",
     "@bedrock/server": "^5.0.0",
     "@bedrock/session-http": "^5.0.0",
     "@bedrock/session-mongodb": "^6.0.0",
     "@bedrock/test": "^8.0.5",
     "@bedrock/validation": "^7.0.0",
-    "@bedrock/web-account": "^5.0.0",
-    "@bedrock/web-authn-token": "^6.0.0",
+    "@bedrock/web-account": "^6.0.0",
+    "@bedrock/web-authn-token": "^7.0.0",
     "@bedrock/web-session": "file:..",
     "@digitalbazaar/totp": "^1.0.1",
+    "@vue/compiler-sfc": "^3.3.4",
```

Comment:
Odd that we needed to add this.

Reply:
Using `@bedrock/karma` version 6 requires us to either have vue >=3.2.13 or @vue/compiler-sfc in the dependency. 
This is the error I am seeing: 

/Users/tashid.gyeltshen/Desktop/DigitalBazaar/bedrock-web-session/test/node_modules/vue-loader/dist/compiler.js:14
        throw new Error(`@vitejs/plugin-vue requires vue (>=3.2.13) or @vue/compiler-sfc ` +
              ^
```
Error: @vitejs/plugin-vue requires vue (>=3.2.13) or @vue/compiler-sfc to be present in the dependency tree.
    at Object.<anonymous> (/Users/tashid.gyeltshen/Desktop/DigitalBazaar/bedrock-web-session/test/node_modules/vue-loader/dist/compiler.js:14:15)
    at Module._compile (node:internal/modules/cjs/loader:1233:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1287:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Module._load (node:internal/modules/cjs/loader:938:12)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:119:18)
    at Object.<anonymous> (/Users/tashid.gyeltshen/Desktop/DigitalBazaar/bedrock-web-session/test/node_modules/vue-loader/dist/index.js:29:20)
    at Module._compile (node:internal/modules/cjs/loader:1233:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1287:10)
```